src/main/java/mate/academy/internetshop/dao/jdbc/ProductDaoImpl.java

```diff
@@ -0,0 +1,118 @@
+package mate.academy.internetshop.dao.jdbc;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import mate.academy.internetshop.dao.ProductDao;
+import mate.academy.internetshop.lib.anno.Dao;
+import mate.academy.internetshop.model.Product;
+import mate.academy.internetshop.util.ConnectionUtil;
+import org.apache.log4j.Logger;
+
+@Dao
+public class ProductDaoImpl implements ProductDao {
+    private static final Logger LOGGER = Logger.getLogger(ConnectionUtil.class);
+
+    @Override
+    public Product create(Product element) {
+        String query = "INSERT INTO internetshop.products(name, price) VALUES (?, ?)";
+
+        try (Connection connection = ConnectionUtil.getConnection()) {
+            PreparedStatement preparedStatement = connection.prepareStatement(query);
+            preparedStatement.setString(1, element.getName());
+            preparedStatement.setDouble(2, element.getPrice());
+            preparedStatement.execute();
+        } catch (SQLException e) {
+            LOGGER.warn("Unable to execute create product query. Stack trace: " + e.getMessage());
```

Comment:
If you you couldn't create Product, your program will continue to work. Is that ok?

Reply:
@dmytro-hryhoruk 

Due to signature of GenericDao.create() method we have to return instance of element, not boolean. Today Bohdan said that's ok, and no need to refactor.
Anyway, I think it is good, that program continue to work without crashes, and if problem causes in this block we can check log file.

Reply:
@dmytro-hryhoruk 

New exception added

Reply:
so eventually you added exception to this method but what about others?